harness/determined/_trial_controller.py

```diff
@@ -1,11 +1,16 @@
 import abc
 import logging
+import os
+import enum
 from typing import Any, Optional, Type
 
 import determined as det
-from determined import horovod, profiler, workload
+from determined import profiler, workload
 from determined.common import check
-from determined.horovod import hvd
+
+
+class _DistributedBackend(enum.Enum):
```

Comment:
Please see the previous comment about this being an enum.  I marked it as unresolved.

Also, please don't mark something as resolved unless you are 100% sure you've fixed it.  IMO it's better not to mark something as resolved until the review has had a chance to see how you've fixed it, then the reviewer can mark it as resolved.  But not everybody agrees with that.

But definitely don't mark something as resolved when it's not fixed.